bundestag.io/dip/src/dip-types.ts

```diff
@@ -64,4 +75,5 @@ export type Vorgangsposition = {
  vorgang_id: string
  datum: Date
  fundstelle: Fundstelle
+ beschlussfassung?: Array<Beschlussfassung>
```

Comment:
```suggestion
 beschlussfassung?: Beschlussfassung[]
```
wenn möglich definiere ich in der Regel arrays so. Find ich etwas übersichtlicher. Fein für dich?

Reply:
klar